rust/log-service/src/lib.rs

```diff
@@ -1075,6 +1075,17 @@ impl LogServer {
                     Status::new(err.code().into(), err.to_string())
                 }
             })?;
+            if let Some(cache) = self.cache.as_ref() {
+                let cache_key = format!("{collection_id}::MANIFEST");
```

Comment:
nit - move cache key creator out so we don't have same-string assumptions across code